core/embed/rust/src/protobuf/obj.rs

```diff
@@ -191,33 +194,38 @@ impl TryFrom<Obj> for Gc<MsgDefObj> {
             let this = unsafe { Gc::from_raw(value.as_ptr().cast()) };
             Ok(this)
         } else {
-            Err(Error::InvalidType)
+            Err(Error::TypeError)
         }
     }
 }
 
 unsafe extern "C" fn msg_def_obj_attr(self_in: Obj, attr: ffi::qstr, dest: *mut Obj) {
     util::try_or_raise(|| {
-        let this=  Gc::<MsgDefObj>::try_from(self_in)?;
+        let this = Gc::<MsgDefObj>::try_from(self_in)?;
         let attr = Qstr::from_u16(attr as _);
 
-        if unsafe { dest.read() } != Obj::const_null() {
-            return Err(Error::InvalidOperation);
+        if !(unsafe { dest.read() }).is_null() {
```

Comment:
Could we split this so it's a bit more readable?

Reply:
fixed in 82f9995f1fa7aa44bf8535345f713925c2f788be